src/command/identity/export.ts

```diff
@@ -0,0 +1,77 @@
+import { Argument, LeafCommand } from 'furious-commander'
+import { exit } from 'process'
+import { Identity, IdentityType } from '../../service/identity/types'
+import { RootCommand } from '../root-command'
+import { printNoIdentitiesError, promptUserForIdentity } from './common'
+
+export class Export extends RootCommand implements LeafCommand {
+  // CLI FIELDS
+
+  public readonly name = 'export'
+
+  public readonly description = 'Export identity'
+
+  @Argument({ key: 'identity-name', describe: 'Name of the identity to be exported' })
+  public identityName!: string
+
+  public async run(): Promise<void> {
+    this.initCommand()
+    this.checkForNoIdentities()
+    const identities = this.getV3Identities()
+    this.checkForEmptyIdentities(identities)
+    await this.ensureIdentityNameIsProvided(identities)
+    this.checkForExistingIdentity()
+    const identity: Identity = this.commandConfig.config.identities[this.identityName]
+    this.checkForV3Identity(identity)
+    this.console.log(JSON.stringify(identity.wallet, null, 4))
+  }
+
+  /** Init additional properties of class, that are not handled by the CLI framework */
+  private initCommand(): void {
+    super.init()
+  }
+
+  private getV3Identities(): string[] {
+    return Object.entries(this.commandConfig.config.identities)
+      .filter(identity => identity[1].identityType === IdentityType.v3)
+      .map(identity => identity[0])
+  }
```

Comment:
Can you give me more context about these indexing on identity like this please?

Reply:
`Object.entries()` returns an array of `[key, value]` pairs. It is the love child of `Object.keys()` and `Object.values()`.

I need both informations:

`[1]` / `value` / _identity object_ - to check `identityType` if it is a V3 wallet
`[0]` / `key` / _name of the identity_ - to collect the possible choices for the end user